spec/requests/v0/login_adoption_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'LoginAdoptions', type: :request do
+  before { sign_in_as(user) }
+
+  describe 'GET v0/login_adoption/index' do
+    context 'when user has a ds logon credential' do
+      let(:user) { create(:user, :dslogon) }
+
+      it 'is a valid request' do
+        get '/v0/login_adoption'
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'returns true for organic adoption modal' do
```

Comment:
You'll also have to add the cases inside this context:
- When the flipper is enabled/disabled
- When the user has an associated AcceptableVerifiedCredential
- When the user has an associated IdmeVerifiedCredential
- When the user does not have associated AVC or IVC